### PR TITLE
Landing Page

### PR DESCRIPTION
**Pull request by dgboss**
Create desktop and mobile friendly landing pages.
The break between mobile and desktop occurs at 600px.

# Test Links:
[Percentile Calculator](https://wps-pr-2531.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-2531.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-2531.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-2531.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-2531.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-2531.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-2531.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

**Comment by conbrad** [comment]
Love it, looks great, just a couple things:

- Is there no tile / sidebar link for HFI Calculator?
- Looks like FireBat links to HFI calculator
- Just need to adjust the links in: https://github.com/bcgov/wps/blob/a5e5f21cb383e387886f16ebc56a2cea4e7b0ea8/.github/workflows/pr_description.js

(Percentile calculator -> `/percentile-calculator`, Landing page -> `/`)

File: web/src/features/landingPage/toolInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    'A tool that supports the calculation of fire behaviour metrics given forecast or actual weather conditions and user-specified fuel types.',
+  icon: <CalculateOutlinedIcon color="primary" fontSize={ICON_FONT_SIZE} />,
+  isBeta: false
+}
+
+export const moreCastInfo: ToolInfo = {
+  name: MORE_CAST_NAME,
+  route: MORECAST_ROUTE,
+  description:
+    'A system that skill scores numerical weather models and enables selection, bias correction, and integration of weather forecast information with others applications.',
```
Comment (andrea-williams) [inline web/src/features/landingPage/toolInfo.tsx:72]:
```suggestion
    'A system that skill scores numerical weather models and enables selection, bias correction, and integration of weather forecast information with other applications.',
```

File: web/src/utils/constants.ts [hunk 2]
```diff
@@ -23,10 +23,22 @@ export const HFI_CALC_ROUTE = '/hfi-calculator'
...
 export const FULL_WIDTH = 1200
 
 // Map center
 export const CENTER_OF_BC = [-123.3656, 51.4484]
+
+// Application names
+export const C_HAINES_NAME = 'C-Haines'
+export const FIRE_BEHAVIOUR_ADVISORY_NAME = 'Auto Spatial Advisory'
+export const FIRE_BEHAVIOUR_CALC_NAME = 'FBP Go'
+export const HFI_CALC_NAME = 'FireBat'
```
Comment (andrea-williams) [inline web/src/utils/constants.ts:39]: FireBat and HFI Calculator are two different applications...

Comment (andrea-williams) [reply web/src/utils/constants.ts:39]: FireBat is at https://psu.nrs.gov.bc.ca/fire-behaviour-calculator
HFI Calculator is at https://psu.nrs.gov.bc.ca/hfi-calculator

**Review by andrea-williams — CHANGES_REQUESTED** [review]

File: web/src/utils/constants.ts (hunk 2 above)
Comment (dgboss) [reply web/src/utils/constants.ts:39]: Haha, yes! Connor also caught this while I blindly followed the mockups. I'm waiting on a response from Neal on going with 7 tiles and keeping FBP Go versus dropping it and sticking with 6 tiles for now.

Comment (dgboss) [reply web/src/utils/constants.ts]: Added FBP Go link to Sybrand's Github page and sorted out HFI Clac and Fire Bat links I think.

Comment (dgboss) [reply web/src/utils/constants.ts]: Waiting on the PO for an HFI description to replace Lorem Ipsum placeholder

**Review by conbrad — APPROVED** [review]

**Review by andrea-williams — APPROVED** [review]
🚀🔥

**Comment by dgboss** [comment]
Just waiting on a few bits and pieces including:
1. HFI Calc description
2. Higher res BC WIldfire logo
3. URLs for links in the footer

**Pull request merged**